src/App.jsx

```diff
@@ -1,10 +1,98 @@
-function App() {
+import { useState } from 'react';
+import { Container, Box, Typography, Button } from '@mui/material';
+import { SearchBar } from './components/SearchBar/SearchBar';
+import { ArticlesList } from './components/ArticlesList/ArticlesList';
+import { ArticleModal } from './components/ArticleModal/ArticleModal';
+import { SortButton } from './components/SortButton/SortButton';
+import useLocalStorage from './hooks/useLocalStorage';
+import useArticles from './hooks/useArticles';
+import useArticleEdit from './hooks/useArticleEdit';
+import useArticleCreate from './hooks/useArticleCreate';
+import useSearchQuery from './hooks/useSearchQuery';
+
+const App = () => {
+  const [searchQuery, setSearchQuery] = useSearchQuery();
```

Comment:
Looks like you want to always use the `searchQuery` and `setSearchQuery` variable names. If that's the case, it makes more sense to return an object from the `useSearchQuery` hook.